packaging/build_deb

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Builds the libsxg deb packages. The script must be run at the root
+# directory of the libsxg source tree. The deb packages will be copied to
+# the current directory.
+#
+# This script is written mainly for CI.
+
+set -ex
+
+readonly SRCDIR="${PWD}"
+readonly WORKDIR="$(mktemp -d --tmpdir=/tmp build_deb.XXXXXX)"
+readonly NAME='libsxg'
+readonly VERSION='0.1'
+readonly TARNAME="${NAME}_${VERSION}"
+readonly TARGET_NAME='libsxg-dev'
+readonly ARCHITECTURE="$(dpkg --print-architecture)"
+
+echo "Working in ${WORKDIR}"
+
+git archive --format=tar.gz --output="${WORKDIR}/${TARNAME}.orig.tar.gz" \
+    --prefix="${TARNAME}/" HEAD
+cd "${WORKDIR}"
+tar xf "${TARNAME}.orig.tar.gz"
+cd "${TARNAME}"
+ln -s "packaging/debian" .
```

Comment:
No need for quotes; we don't usually quote simple strings without spaces, especially filenames.

If you _really_ want quotes, you should use single quotes. [Shell Style Guide](https://google.github.io/styleguide/shell.xml?showall=y#Quoting)

Reply:
changed to `'`.